packages/nhsuk-frontend/src/nhsuk/core/elements/_links.scss

```diff
@@ -26,16 +26,37 @@ a {
   }
 }
 
-.nhsuk-link--no-visited-state {
-  @include nhsuk-link-style-no-visited-state;
+// Variant classes should always be used in conjunction with the .nhsuk-link
```

Comment:
Currently the `.nhsuk-link` class doesn’t actually do anything at all, and we rely on the `a { }` selector above?

Should we either change it to `a, .nhsuk-link {}` - or leave it and update this comment?

Reply:
Updated in https://github.com/nhsuk/nhsuk-frontend/pull/1367/commits/7014be0ff7e088f9ea83a781752b16ce005d2fee